### PR TITLE
Question and Result feature

### PR DESCRIPTION
**Pull request by cjinc121**
added Question M.C.Q with timer 
added Result Page
Screenshot (181)
Screenshot (180)

### CODE REVIEW

File: client/src/App.tsx [hunk 1]
```diff
@@ -1,15 +1,20 @@
...
+      <Header />
       <Routes>
-        <Route path="/"element={<Homepage/>}></Route>
-        <Route path="/:quizName/:questionNo" element={<QuizQuestionPage/>}></Route>
+        <Route path="/" element={<Homepage />}></Route>
+        <Route
+          path="/quiz/:quizName/:questionNo"
+          element={<QuizQuestionPage />}
+        ></Route>
+        <Route path="/quiz/:quizName/result" element={<Result />}></Route>
```
Comment (ShreyashMahajan) [inline client/src/App.tsx:17]: you can create a separate file for routes

**Review by ShreyashMahajan — APPROVED** [review]
Code is well structured, 
naming is easily understandable,
good to merge

File: client/src/components/QuestionCard/QuestionCard.tsx [hunk 2]
```diff
@@ -1,42 +1,114 @@
...
-<div className="options-container">
-<div className="option">1.Pakistan</div>
-<div className="option">2.INDIA</div>
-<div className="option">3.UNITED STATES OF AMERICA</div>
-<div className="option">4.AUSTRALIA</div>
-</div>
-</div>
-}
\ No newline at end of file
+export const QuestionCard = ({
```
Comment (ArjunGTX) [inline client/src/components/QuestionCard/QuestionCard.tsx:11]: As a rule of thumb, it is better to use an interface for Props

**Review by ArjunGTX — APPROVED** [review]

**Review by ajit-sharmaGH — APPROVED** [review]
Good work on Ui, you can merge it, everything looks fine

**Pull request merged**